src/test/resources/simple/not-allowed-url-bag/fetch.txt

```diff
@@ -0,0 +1 @@
+ftp://dans.knaw.nl/nl/over/organisatie-beleid/informatiemateriaal/Software_Sustainability_DANS_NLeSC_2016.pdf  555292  data/brochure/Software_Sustainability_DANS_NLeSC_2016.pdf
```

Comment:
For a similar case in the sword2-examples project we have uploaded a couple of files to GitHub. Maybe you could do the same for this project.